google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcStorageImpl.java

```diff
@@ -159,11 +160,15 @@ final class GrpcStorageImpl extends BaseService<StorageOptions> implements Stora
   final GrpcRetryAlgorithmManager retryAlgorithmManager;
   final SyntaxDecoders syntaxDecoders;
 
+  // workaround for https://github.com/googleapis/java-storage/issues/1736
+  private final Opts<UserProject> defaultOpts;
```

Comment:
defaultOpts will only ever contain a single userProject correct? it seems confusing to call it defaultOpts alluding that it may have more than one value. Unless we are assuming we might have to do this again?

Reply:
`Opts` itself is a collection, so there can theoretically be multiple UserProject instances in it, it felt weird to name it `defaultUserProject` when from the standpoint of the usage of this value the operations themselves don't really care about it being UserProject just that it can be prepended as a lower priority in case a particular `Opt` isn't already defined.